src/udev/udev-node.c

```diff
@@ -80,7 +80,7 @@ static int node_symlink(sd_device *dev, const char *devnode, const char *slink)
         if (!devnode) {
                 r = sd_device_get_devname(dev, &devnode);
                 if (r < 0)
-                        return log_device_debug_errno(dev, r, "Failed to get device node: %m");
+                        return log_device_error_errno(dev, r, "Failed to get device node: %m");
```

Comment:
About the first commit:
I do not think it is necessary to upgrade the log level. All failures are already logged in error or warning level in the caller side, and the caller side logs already explained well what operation, that is creating/removing symlink, failed. Logging e.g. `Failed to get device node` is not clear about what is caused by the failure.

If you really do like that, then please drop the logs in the caller side, and update each log message.